Sources/SwiftInspectorVisitors/TypeInheritanceVisitor.swift

```diff
@@ -30,7 +30,7 @@ public final class TypeInheritanceVisitor: SyntaxVisitor {
   public private(set) var inheritsFromTypes = [String]()
 
   public override func visit(_ node: InheritedTypeSyntax) -> SyntaxVisitorContinueKind {
-    inheritsFromTypes.append(node.typeName.description.trimmingCharacters(in: .whitespacesAndNewlines))
+    inheritsFromTypes.append(node.typeName.qualifiedName)
```

Comment:
These two lines are functionally equivalent

Reply:
To clarify though... we are no longer stripping whitespace. But that is OK since we're pulling the name from a more robust place now?

Reply:
yup! and our unit tests prove that out 🙂